generic/tests/netperf.py

```diff
@@ -141,6 +137,27 @@ def mtu_set(mtu):
             process.run(host_mtu_cmd % (iface, mtu), ignore_status=False,
                         shell=True)
 
+    def tweak_tuned_profile():
+        """
+
+        Tweak configuration with truned profile
+
+        """
+
+        client_tuned_profile = params.get("client_tuned_profile")
+        server_tuned_profile = params.get("server_tuned_profile")
+        host_tuned_profile = params.get("host_tuned_profile")
+        error_context.context("Changing tune profile of guest", logging.info)
+        if params.get("os_type") == "linux" and server_tuned_profile:
+            ssh_cmd(server_ctl, server_tuned_profile)
+
+        error_context.context("Changing tune profile of client/host",
+                              logging.info)
+        if client_tuned_profile:
+            ssh_cmd(client, client_tuned_profile)
+        if host_tuned_profile:
+            ssh_cmd(host, host_tuned_profile, ignore_status=False)
+
```

Comment:
Remove "ignore_status=False" for L159, it is default.

Reply:
I have updated, please help review. thanks